cloud/deployment/deployment.go

```diff
@@ -692,3 +692,20 @@ func deploymentSelectionProcess(ws string, deployments []astro.Deployment, clien
 	}
 	return currentDeployment, nil
 }
+
+// GetDeploymentURL takes a deploymentID, WorkspaceID as parameters
+// and returns a deploymentURL
+func GetDeploymentURL(deploymentID, workspaceID string) (string, error) {
+	var (
+		deploymentURL string
+		ctx           config.Context
+		err           error
+	)
+
+	ctx, err = config.GetCurrentContext()
+	if err != nil {
+		return "", err
+	}
+	deploymentURL = "cloud." + ctx.Domain + "/" + workspaceID + "/deployments/" + deploymentID + "/analytics"
```

Comment:
Do we need add extra support to generate this for pr-preview environments?

Reply:
yes we need to add support for pr previews and it is tracked in #744